integrations/wpseo/wpseo.php

```diff
@@ -265,9 +268,6 @@ public function add_post_type_archive( $str ) {
 			$slug = ( true === $post_type_obj->has_archive ) ? $post_type_obj->rewrite['slug'] : $post_type_obj->has_archive;
 
 			if ( ! wpcom_vip_get_page_by_path( $slug ) ) {
-				// The post type archive in the current language is already added by WPSEO.
-				$languages = wp_list_filter( $languages, array( 'slug' => pll_current_language() ), 'NOT' );
-
```

Comment:
Must not be removed: we want to keep the languages that are not the CURRENT language (!== default).

Reply:
Maybe I missed something, but is it done in line 252?

Reply:
As we discussed, everything has been reverted for this file because we're dealing with the default language on one place and with the current language on the other place.